src/molecules/Dialog/Dialog.tsx

```diff
@@ -65,13 +73,30 @@ export const Dialog = forwardRef<HTMLDivElement, DialogProps>(
       children,
       width,
       actions,
-      withContentPadding = true,
+      withContentPaddingX = true,
+      withContentPaddingY = true,
```

Comment:
Prefer to have these as 1 prop like this:
```
  withContentPadding: {
     height: true,
     width: true,
  }
```

Reply:
Changed to reflect that pattern